eisbuk-admin/src/store/selectors/__testData__/customers.ts

```diff
@@ -0,0 +1,38 @@
+import { LocalStore } from "@/types/store";
+
+import { createTestStore } from "@/__testUtils__/firestore";
+
+import { saul, walt, jian, mike, jane } from "@/__testData__/customers";
+
+/**
+ * A customers store input
+ */
+const customers: LocalStore["firestore"]["data"]["customers"] = {
+  [saul.id]: saul,
+  [jian.id]: jian,
+  [walt.id]: walt,
+  [mike.id]: mike,
+  [jane.id]: jane,
+};
+
+export const expectedCustomersBirthdays = [
+  {
+    birthday: "2009-12-23",
```

Comment:
We're expecting "12-23" to be the birthday as customers born on "2021-12-23" and "1994-12-23" will have the same birthday